bouncer/shared/utils.ts

```diff
@@ -221,6 +221,15 @@ function getCachedSubstrateApi(defaultEndpoint: string) {
     api = await ApiPromise.create({
       provider: new WsProvider(endpoint),
       noInitWarn: true,
+      types: {
+        EncodedAddress: {
+          _enum: {
+            Eth: '[u8; 20]',
+            Dot: '[u8; 32]',
+            Btc: '[u8; 34]',
```

Comment:
I don't think we can guarantee that bitcoin addresses are always 34 bytes (and in fact the encoding is a Vec<u8> which is probably not compatible with [u8; 34] encoding) - is there another way to express this? For example `Vec<u8>` or `String` or something equivalent?